src/options/optionsMain.ts

```diff
@@ -1,3 +1,5 @@
+import axios from 'axios';
+
 import * as fs from 'fs';
 
 import * as log from 'loglevel';
```

Comment:
Nit: in many languages, it's conventionally common to order imports in the following order:

1. Standard library imports (here, `fs`)
2. Third-party library imports (here, `axios` and `loglevel`)
3. Internal imports

, each block ordered alphabetically and each block separated by a whitespace. So,

```suggestion
import * as fs from 'fs';

import axios from 'axios';
import * as log from 'loglevel';
```